src/Windows/Avalonia.Win32/ClipboardFormatRegistry.cs

```diff
@@ -52,6 +56,32 @@ public static ushort GetFormatId(DataFormat format)
         {
             lock (s_formats)
             {
+                if (DataFormat.Bitmap.Equals(format))
+                {
+                    (DataFormat, ushort)? pngFormat = null;
+                    (DataFormat, ushort)? jpgFormat = null;
+                    (DataFormat, ushort)? dibFormat = null;
+                    (DataFormat, ushort)? bitFormat = null;
+
+                    foreach (var currentFormat in s_formats)
+                    {
+                        if (currentFormat.Id == (ushort)UnmanagedMethods.ClipboardFormat.CF_DIB)
+                            dibFormat = currentFormat;
+                        if (currentFormat.Id == (ushort)UnmanagedMethods.ClipboardFormat.CF_BITMAP)
```

Comment:
Nit: `else` missing on this `if` and all subsequent ones (I can't find my initial comment about this?).